BadHabbit/BadHabbit/XCAssets+Generated.swift

```diff
@@ -0,0 +1,105 @@
+// swiftlint:disable all
```

Comment:
Add generated files from SwiftGen to gitignore

Reply:
Generated files added into .gitignore